File: app/src/main/java/com/luffy/mulmedia/gles2/IDrawer.java
package com.luffy.mulmedia.gles2;

public interface IDrawer {
    void draw();

    void setTextureId(int[] id);

    void release();

    void setSurfaceSize(int w, int h);

    void setVideoSize(int w, int h);

    void translate(float translateX, float translateY);

    void scale(float scaleX, float scaleY);

    void setShader(IGLShader shader);
}

File: opengles-base/src/main/java/com/luffyxu/opengles/base/egl/IDrawer.kt
package com.luffyxu.opengles.base.egl

interface IDrawer {
    fun draw()
    fun setTextureId(id: IntArray)
    fun release()
    fun setSurfaceSize(w: Int, h: Int)
    fun setVideoSize(w: Int, h: Int)
    fun translate(translateX: Float, translateY: Float)
    fun scale(scaleX: Float, scaleY: Float)
    fun setShader(shader: IGLShader)
}